Etiquette/CODE_REVIEW.md

```diff
@@ -0,0 +1,110 @@
+* Be grateful for the reviewer's suggestions. ("Good call. I'll make that
+  change.")
+* If you disagree with a suggestion, explain why and avoid template answers like "out of the scope" or "thank you for your opinion". If you think that suggestion is a personal preference and does not bring value - say so
+* Assume the best intention from the reviewer's comments.
+* If a reviewer seems aggressive or angry or otherwise personal, consider if it is intended to be read that way and ask the person for clarification of intent, in person if possible. Do not escalate online discussions.
+* If you can't reach agreement seek for more opinions
+* Don't be too defensive about your code, someone will rewrite it anyway sooner or later (it might be yourself as well)
+* Try to answer all comments, don't make your colleagues feel like you are ignoring them. You can review your own code for that instead of sending individual replies so that all your answers are posted at once.
+* Add "work in progress" label while addressing review comments
+* Re-request review when you addressed the comments (either implementing suggested change or replying to the comment) and label it with "Ready for review" again
+* Do not rebase and force-push to your pull request branch after you have received any comments, use merge instead. This allows reviewers to view changes on the PR since their last review, rather than re-reviewing a lot of the code they have already reviewed. All pull requests are squashed when they are merged into the base branch so the history of your changes won't matter. It's fine to rebase and force-push if there are no comments.
+* Do not dismiss any pending requests for changes unless you agreed with their authors that it was addressed
+* When you have two approvals and have no pending questions or requests for changes and don't wait for review from anyone else - put a "Merge" label for bot to merge your changes.
+
+
+PR Parties
+--------
+
+To improve collaboration, we take time to review pull request together in small groups every week. This is what we call "PR parties".
+
+- Each group should include at least one senior developer. 
+- Each group will have a few pull requests assigned to them
+- When done with their assigned PRs, the group is free to review something else (i.e. each other's pull requests) or finish the meeting
+
+Pull Assigners
+--------
+
+To balance the workload and avoid knowledge silos we use [Pull Assigners](https://pullpanda.com/assigner) to assign reviewers.
+
+- Each pull request will be assigned to three random reviewers from the team (PS: if the PullAssigners bot seems stuck, try to de-assign it and re-assign it.)
+- Two approvals are enough to merge a pull request (there shouldn't be pending requests for changes though and you should address any other pending questions or comments)
+- Anyone else is still encouraged to review any pull request and author can request reviews from specific teammates
```

Comment:
What happens if a PR has two approvals but the author has requested review from a specific teammate that hasn't reviewed it yet? Shouldn't in this case the author wait for that specific review?

Reply:
yep, hence that's up to the author to put merge label (I think that's mentioned somewhere in the doc, will check that)

Reply:
Yes, you are right, it is mentioned above. Leave it then 😊